circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
```

Comment:
This looks like CircleCI 1.0, right? CircleCI 1.0 is being sunset in August this year. I think starting with CircleCI 2.0 might be worth it considering 1.0 will stop working relatively soon.